README.md

```diff
@@ -22,7 +22,7 @@ Linux Kernel is treated as the authorative eBPF implementation.
 ### Ubuntu
 
 ```
-sudo apt-get install -y libboost-dev libboost-filesystem-dev libboost-program-options-dev libelf-dev lcov
+sudo apt-get install -y libboost-dev libboost-filesystem-dev libboost-program-options-dev libelf-dev lcov libbpf-dev
```

Comment:
What is this needed for? I think we download and build libbpf v1.3.0, so why would we need the old version provided by Ubuntu?

Reply:
There's nothing in the list of prerequisites for users that tell them to download and build libbpf.
Should there be?

Reply:
Got it. This is the README not the CI/CD. My bad I was confused.